### DISCUSSION TITLE
Am I doing something wrong with targeting?

### DISCUSSION THREAD

**Discussion by dreamforge2361** [wildj79/foundryvtt-starfinder#1282] [Q&A] [answered]
When I take two randomn NPC's and put them on a map to test out the system.  I start a combat encounter, and then I have one target the other one and use one of its attacks or weapons, but it doesn't auto-calculate anything, the item card in the chat, shows the attack or damage and it rolls, but it doesn't check against AC or KAC or any of that and it doesn't auto apply damage or anything?  Am I doing something wrong or are those features not in the system yet?

**Comment by ian612** [comment, marked as answer]
Hi there, these features are not in the system. The GM needs to check and verify the rolled value against AC manually. Damage can be quickly applied by right clicking on a damage/healing chat card, though. There is a menu that pops up that allows it to be applied to the currently selected (not targeted) token.
